website/src/pages/api/auth/[...nextauth].ts

```diff
@@ -44,6 +44,7 @@ if (boolean(process.env.DEBUG_LOGIN) || process.env.NODE_ENV === "development")
         const user = {
           id: credentials.username,
           name: credentials.username,
+          role: "admin",
```

Comment:
This seems unrelated.  Is it needed?  the `.env` is configured to set the `admin` debug user with the `admin` role.  We didn't document that anywhere but it should work.